### ISSUE TITLE
Submit endpoint seems to hang when overwrite_merge==3

### ISSUE THREAD

**Issue by bbartley** [SynBioHub/synbiohub#1126]
The submission endpoint called by pySBOL was changed from `remoteSubmit` to `submit`.  Now, when running integration tests between pysbol/synbiohub, overwrite_merge modes 0, 1, and 2 execute successfully but overwrite_merge==3 seems to hang.

**Comment by cjmyers** [comment]
I'm unable to reproduce this using either the browser or libSBOLj.  Can you provide more details about the test you are running?

**Comment by bbartley** [comment]
The test in simple terms is as follows.  Create a `ComponentDefinition` and assign a `role` as an SO term.  Push this to a `Collection`.  Now change the `role` to a new SO term and attempt to merge and overwrite the `Collection`.

Here is the longform Python code:
```
from sbol import *
from urllib3.exceptions import HTTPError

RESOURCE = 'https://hub-staging.sd2e.org'
SPOOFED_RESOURCE = 'https://hub.sd2e.org'
USER = <user>
PASSWORD = <password>
PART_SHOP = PartShop(RESOURCE)
if SPOOFED_RESOURCE:
    PART_SHOP.spoof(SPOOFED_RESOURCE)
PART_SHOP.login(USER, PASSWORD)

TEST_COLLECTION = 'pySBOL_test'
TEST_COLLECTION_URI = RESOURCE + '/user/' + USER + '/' + TEST_COLLECTION + '/' + TEST_COLLECTION + '_collection/1'
Config.setOption('sbol_typed_uris', False)

# test submit
print('Testing submit...', end = '')
doc = Document()
doc.displayId = TEST_COLLECTION
doc.name = 'pySBOL test'
doc.description = 'A temporary collection used for running pySBOL integration tests'
doc.componentDefinitions.create('foo')
PART_SHOP.submit(doc)
print('SUCCESS')

# test submit -- overwrite
print('Testing submit overwrite...', end = '')
doc = Document()
doc.displayId = TEST_COLLECTION
doc.name = 'pySBOL test'
doc.description = 'A temporary collection used for running pySBOL integration tests'
bar = doc.componentDefinitions.create('bar')
bar.roles = SO_PROMOTER
PART_SHOP.submit(doc, TEST_COLLECTION_URI, 1)
print('SUCCESS')

# test submit -- merge and prevent
print('Testing merge and prevent...', end = '')
doc = Document()
doc.displayId = TEST_COLLECTION
doc.name = 'pySBOL test'
doc.description = 'A temporary collection used for running pySBOL integration tests'
bar = doc.componentDefinitions.create('bar')

# Change bar's roles field from what it was set to in test_submit_overwrite
bar.roles = SO_CDS
try:
    PART_SHOP.submit(doc, TEST_COLLECTION_URI, 2)
except HTTPError:
    print('SUCCESS')

# test submit -- merge and overwrite
print('Testing merge overwrite...', end = '')
doc = Document()
doc.displayId = TEST_COLLECTION
doc.name = 'pySBOL test'
doc.description = 'A temporary collection used for running pySBOL integration tests'
bar = doc.componentDefinitions.create('bar')

# Change bar's roles field from what it was set to in test_submit_overwrite
bar.roles = SO_CDS
PART_SHOP.submit(doc, TEST_COLLECTION_URI, 3)            
print('SUCCESS')
```

**Comment by cjmyers** [comment]
Here is my test:

		SBOLReader.setURIPrefix("http://www.dummy.org");
		SBOLDocument docS = SBOLReader.read("/Users/myers/Downloads/sequence.gb");
		SynBioHubFrontend sbh2 = new SynBioHubFrontend("http://localhost:7777","https://synbiohub.org");
		sbh2.login("myers@ece.utah.edu", "test");
		sbh2.createCollection("testAPI", "1", "testAPI", "testAPI", "", true, docS);
		sbh2.addToCollection(URI.create("https://synbiohub.org/user/myers/testAPI/testAPI_collection/1"), true, docS);
		
I see later if I can mimic yours completely in Java.  Note that the API in Java.  Please don’t use 0, 1, 2, 3 in your code.  This is not guaranteed to be maintained. The API really about create/add and prevent/overwrite.  Please refactor your API towards this more intuitive one, and deprecate the other.  Thanks.

**Comment by cjmyers** [comment]
I was able to reproduce.  Key was actually overwriting and merging into existing collection with an object with new content.  Fixed in PR #1129
